[PAN-3134] Add retesteth subcommand. (#4)
hyperledger/besu-docs: docs/Reference/CLI/CLI-Subcommands.md

        
        ``` tab="RLP Output"
        0xf853a00000000000000000000000000000000000000000000000000000000000000000ea94be068f726a13c8d46c44be6ce9d275600e1735a4945ff6f4b66a46a2b2310a6f3a93aaddc0d9a1c193808400000000c0
        ```

## retesteth

> Runs a Retesteth compatible server. [Retesteth](https://github.com/ethereum/retesteth/wiki) is a developer tool that runs Ethereum consensus tests on an Ethereum client.

Reviewer: Maybe worth adding a note that it also allows authoring of consensus tests?  
```suggestion
Runs a Retesteth-compatible server. [Retesteth](https://github.com/ethereum/retesteth/wiki) is a developer tool that can generate and run consensus tests against any Ethereum client running such a server.
```

        
        ``` tab="RLP Output"
        0xf853a00000000000000000000000000000000000000000000000000000000000000000ea94be068f726a13c8d46c44be6ce9d275600e1735a4945ff6f4b66a46a2b2310a6f3a93aaddc0d9a1c193808400000000c0
        ```

## retesteth

> This command enables Retesteth-compatibilty, and accepts the following command line options:

Reviewer 2: ```suggestion
Runs a Retesteth compatible server.  This command accepts the following command line options:
```

I think the first _this command_ is redundant and splitting into two sentences simplifies the language.

* [\--data-path](../CLI-Syntax#data-path)
* [\--host-whitelist](../CLI-Syntax#host-whitelist)
* [\--rpc-http-host](../CLI-Syntax#rpc-http-host)
* [\--rpc-http-port](../CLI-Syntax#rpc-http-port)
* [\--logging](../CLI-Syntax#logging)

> !!! note

Reviewer 2: I'd move this into the first paragraph (ie, after _Runs a Retesteth compatible server_) - mainly because I think it's information that's relevant to understanding why you'd use it rather than a side note.

>     [Retesteth](https://github.com/ethereum/retesteth/wiki) is a developer tool that runs Ethereum consensus tests on a client. 

Reviewer 2: ```suggestion
    [Retesteth](https://github.com/ethereum/retesteth/wiki) is a developer tool that runs Ethereum consensus tests on an Ethereum client. 
```

Suggested change to be more specific
